openbas-model/src/main/java/io/openbas/database/model/Group.java

```diff
@@ -69,6 +71,7 @@ public class Group implements Base {
   @Fetch(value = FetchMode.SUBSELECT)
   private List<Grant> grants = new ArrayList<>();
 
+  @ArraySchema(schema = @Schema(type = "string"))
```

Comment:
you should remove from it because FetchType is EAGER

Reply:
I am not sure to understand.
FetchType is EAGER but we still returning only IDs